packages/orchestrator-ui-components/src/messages/en-US.json

```diff
@@ -70,6 +70,10 @@
                 "untaggedPortInUse": "This service port is already in use and cannot be chosen",
                 "taggedOnly": "VLAN is only relevant for SN7 MSP or SN8 SP in tagged mode, not for link_member or untagged ports.",
                 "loadingIms": "Loading VLAN info from IMS"
+            },
+            "subscription": {
```

Comment:
Backend? (probably not, so ignore.)

The `loadingIms` is also very SURF specific. I know that's not your doing.

Reply:
I propose to fix this later when we are separating the surf app